src/Microsoft.Diagnostics.Monitoring.RestServer/Controllers/DiagControllerExtensions.cs

```diff
@@ -78,5 +91,17 @@ public static ObjectResult Problem(this ControllerBase controller, Exception ex)
         {
             return controller.BadRequest(ex.ToProblemDetails((int)HttpStatusCode.BadRequest));
         }
+
+        private static bool LogError(ILogger logger, Exception ex)
+        {
+            logger.LogError(ex, ExceptionLogMessage);
```

Comment:
Will the exception message be visible in the error log here?

Reply:
Yes it will.